### DISCUSSION TITLE
SDL Passolo Text Format

### DISCUSSION THREAD

**Discussion by ianathompson** [WeblateOrg/weblate#5331] [Q&A] [answered]
Any idea where the SDL Passolo Text Format would fit for type of file?
Would we need to build a new file type processor?

https://drive.google.com/file/d/1-cb7Qj7z923izGk9C7HrRWy1EOHzdCI-/view?usp=sharing

Edit1: Code or Quote wouldn't override the Markdown language.

**Comment by nijel** [comment, marked as answer]
It is definitely not supported right now. The support we would need parser and serializer in the translate-toolkit first.

You can file an issue for adding support for that in https://github.com/translate/translate/issues, please include links to the specification of the format there (I hope it exists).

Honestly, I don't think anybody will volunteer to implement this. If you need this feature, and you are unable to contribute the code, you should consider funding the development, see https://weblate.org/en/support/
